services/web-app/components/catalog-multislect-filter/catalog-multiselect-filter.module.scss

```diff
@@ -0,0 +1,120 @@
+//
+// Copyright IBM Corp. 2021, 2021
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+.container {
+  width: 100%;
+  height: $spacing-09;
+  border: none;
+  border-left: 1px solid $border-subtle;
+  background: $background;
+  box-shadow: inset 0 -1px 0 $border-strong;
+  cursor: pointer;
+
+  &:focus {
```

Comment:
Notice how the component jumps a couple pixels when you click it? That's because you're adding a border that didn't previously exist. Look at how we're doing the focus state in the catalog item. You can use outline with a negative offset. Outline doesn't take up space in the box model, and the negative offset moves it inside the component. Also, we have a token for focus color.

```scss
  &:active {
    outline: 2px solid $focus;
    outline-offset: -2px;
  }
```